.gitignore

```diff
@@ -39,3 +39,4 @@ _sandbox
 
 # Android Studio captures folder
 captures/
+tools/iosched-codestyle.xml
```

Comment:
This file is not supposed to be ignored. We check it in so that others (like yourself) can use the same code style when contributing to the project. Please revert this change and revert the changes to that file.

Reply:
This line still needs to be removed.